### ISSUE TITLE
部分twitter博主路由无法获取最新的推文

### ISSUE THREAD

**Issue by smoothan** [DIYgod/RSSHub#12283]
### Routes

```routes
/twitter/user/:id/:routeParams?
```

### Full routes

```fullroutes
/twitter/user/nanaModeltt/excludeReplies=true&includeRts=false
/twitter/user/2doudoudou/excludeReplies=1&includeRts=0
/twitter/user/Lovely_koyoshi/excludeReplies=1&includeRts=false
/twitter/user/WANIMAL912/excludeReplies=1&includeRts=0
```

### Related documentation

https://docs.rsshub.app/social-media.html#twitter

### What is expected?

请帮忙查看是什么原因并修复它

### What is actually happening?

从4月8号左右开始无法获取部分twitter博主最新的推文，如果推文中带图片也无法获取。博主也未开启保护推文的仅关注者查看。尝试使用rsshub.app和自建的rsshub均出现相同情况。

### Deployment information

RSSHub demo (https://rsshub.app)

### Additional info

```shell
NA
```

### This is not a duplicated issue

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this bug has not already been reported

**Comment by Mendelrock** [comment]
Is there a known workaround for RSSHub? I've recently noticed that behaviour too on my self-hosted instance.

**Comment by JamYiz** [comment]
缺失了很多推文，这个问题解决了吗？

**Comment by TonyRL** [comment]
Closed in 9032153c7de4c9ca189482d495696986e9795106
